manifests/0007_10_cluster-network-operator_03_deployment.yaml

```diff
@@ -28,6 +28,12 @@ spec:
         env:
         - name: SDN_IMAGE
           value: "docker.io/openshift/origin-node:v4.0.0"
+        - name: NODE_IMAGE
+          value: "docker.io/openshift/origin-node:v4.0.0-local"
```

Comment:
Before this is merged, we should get rid of the `-local` and `latest` image references.